content/departments/engineering/design/goals/index.md

```diff
@@ -0,0 +1,67 @@
+# Design team goals
+
+We have team-level design team goals to help us align and prioritize our
+focus as a team over the following 3-6 months.
+
+Our team-level goals cascade from the overall company [product vision and
+strategy](../../index.md#product-vision-and-strategy), and each member of the design team
+uses these to define their own individual goals.
+
+## (Draft) Design team goals for the next 3-6 months
+
+### Urgent and important
+
+#### Theme: Win the top 100 organizations
+
+1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
+   - Key Result: Eliminate all the UI/UX-related must-haves from the scale
+     audit.
+   - Key Result: Design as a function contributes to closing a deal for a
```

Comment:
How might we measure this? It could be closing out an issue(s) that is an improvement/feature request that is directly requested by a 2XL customer, but this would be reactive and requires that a customer preemptively requests something that is then tagged as needing design.

Reply:
This is intended to be directly tied to the strategic work underway. Assuming we have a specific list of issues (@AlicjaSuska), then measuring it would be tied to addressing these issues.

I'm thinking that as an objective/key result, it should more specifically align it to the strategic effort?

Reply:
One additional fly-by note: Yes, @AlicjaSuska's work is absolutely critical path for strategic. In addition,  every team across the org is working to increase the strategic readiness of their features and design support will be necessary alongside each of those teams as well. For example, BC team will certainly need thoughtful design as they work on the key issues outlined in their [strat readiness doc](https://docs.google.com/document/d/1vXAnhui-AGBk2aar4pXt-uTuTIQfZU5LFFtrMAU2e4s/edit#bookmark=id.9gi05kl6on12) that fall outside of Alicja's central work.

Reply:
Maybe we could include this in the first KR:
Key Result: Eliminate all the UI/UX-related must-haves from the scale audit.
Key result: Provide design support for all the key issues outlined in the strategic readiness doc for respective teams.

Reply:
I altered this with a slightly edited version of Alicja's comment and will remove the last KR as I think it's a bridge too far and would be hard to measure. If anyone disagrees, feel free to bring it back.